node-commons/src/main/java/org/apache/iotdb/commons/partition/TimePartitionSlot.java

```diff
@@ -18,14 +18,43 @@
  */
 package org.apache.iotdb.commons.partition;
 
+import java.nio.ByteBuffer;
+
 public class TimePartitionSlot {
   private long startTime;
 
+  public TimePartitionSlot() {
+    // empty constructor
+  }
+
+  public TimePartitionSlot(long startTime) {
+    this.startTime = startTime;
+  }
+
   public long getStartTime() {
     return startTime;
   }
 
   public void setStartTime(long startTime) {
     this.startTime = startTime;
   }
+
+  @Override
+  public int hashCode() {
+    return new Long(this.startTime).hashCode();
+  }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
Do not write these two functions by hand, they are very error prone, use IDEA to generate

Reply:
fixed.